### PR TITLE
FIX #6648 - We add a task in RoboFile.php for cleaning cache directory

### LINKED ISSUE #6648: Add task to clean cache dir to RoboFile.php
#### Issue

This issue starts thanks to the debate in #6605 where we conclude that it is better to use [Robo](https://robo.li/) instead of [Makefile](https://www.gnu.org/software/make/manual/make.html)

We should have common and useful tasks that can be runned from the console, like for example a task to clean cache directory.

#### Expected Behavior

It would be great to clean cache directory by running

`$ ./vendor/bin/robo clean:cache`

#### Actual Behavior

We do not have a way to clean cache directory from the console.

### PR DESCRIPTION
**Pull request by Abuelodelanada**
## Description

See #6648 

## How To Test This

You can run the new task:

`$ ./vendor/bin/robo clean:cache`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
A long time ago I came across this article

https://academe.co.uk/2012/06/sugarcrm-cache-directory-it-is-not-a-cache-directory/

and since then I never cleared my cache directory completely. I always clean some subdirectories only.

Some of these things have changed, from the SugarCRM days, but I'm not sure what the current status is...

**Comment by Abuelodelanada** [comment]
You are right @pgr!

I remember in a project MakeFile we have this:


```
clean_cache cc:
	rm -rf cache/xml/*
	rm -rf cache/import/*
	rm -rf cache/dashlets/*
	rm -rf cache/modules/*
	rm -rf cache/smarty/*
	rm -rf cache/jsLanguage/*
	rm -rf cache/themes/*
	rm -rf cache/Relationships/*
	rm -rf cache/include/javascript/*
```

I will modify this PR.

**Comment by Abuelodelanada** [comment]
Hi @pgorod 

I made some changes to the task

**Comment by Abuelodelanada** [comment]
Oh... I'm deleting cache/modules/emails ... give me a moment to solve it

File: RoboFile.php [hunk 1]
```diff
@@ -6,5 +6,48 @@
...
+                               'cache/import',
+                               'cache/include/javascript',
+                               'cache/jsLanguage',
+                               'cache/pdf',
+                               'cache/themes',
+                               'cache/xml',
+        );
+
+        foreach ($cacheToDelete as  $dir) {
+            if ((file_exists($dir) && is_dir($dir))) {
```
Comment (Dillon-Brown) [inline RoboFile.php]: You have a redundant parentheses here.

**Comment by Abuelodelanada** [comment]
Hi @Dillon-Brown !
Great! 
In a few hours I'll move the task to ` lib/Robo/Plugin/Commands` and commit to this PR.

**Comment by Abuelodelanada** [comment]
@Dillon-Brown It's done... I moved the command to lib/Robo/Plugins/Commands/

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**